neard/src/cli.rs

```diff
@@ -215,7 +215,6 @@ impl RunCmd {
     pub(super) fn run(self, home_dir: &Path) {
         // Load configs from home.
         let mut near_config = nearcore::config::load_config_without_genesis_records(home_dir);
-        nearcore::genesis_validate::validate_genesis(&near_config.genesis);
```

Comment:
Great removal! The conceptual distance between this call and the place where the fn is defined is large!